### DISCUSSION TITLE
Brightness increases before reducing to the set amount

### DISCUSSION THREAD

**Discussion by 4rshdeep** [MonitorControl/MonitorControl#1092] [General]
### Before opening the issue, have you...?

- [X] Searched for existing issues
- [X] Looked through [the wiki](https://github.com/MonitorControl/MonitorControl/wiki)
- [X] Updated MonitorControl to the latest version (if applicable)

### Describe the bug

Firstly thanks for this extension, it works so good and has made my life so much better.
While using this, when I switch windows (Mac swipe to switch between full windows) brightness increases to the max before reducing which leads to an uncomfortable second or so. 

### Steps to reproduce

Just swipe to switch full screen apps full-screen

### Expected behavior

Brightness should keep the same?

### Environment Information (please complete the following information)

```markdown
- macOS version: 12.4
- Mac model: MacBook Pro (16-inch, 2021)
- MonitorControl version: 4.1.0 Build 7034
- Monitor(s): MacBook Pro (16-inch, 2021)
- Apple Silicon/M1 (yes or no): yes
```

**Comment by waydabber** [comment]
This happens when you are using overlay dimming and disable color table manipulation (for DisplayLink support or compatiblity with f.lux). This is an expected behavior and there is no workaround for this. You can change this under Preferences/Displays.

Hope this info helps! :)
